docs/resources/aom_component.md

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Application Operations Management (AOM)"
+---
+
+# huaweicloud_aom_component
+
+Manages an AOM component resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "topic_urn" {}
+
+resource "huaweicloud_aom_component" "test" {
+  description        = "component description"
+  model_id           = "%s"
```

Comment:
it is not suitable, you can use a variable for replace, for example:
model_id = var.model_id, and the valiable of model_id should be declared